src/routes/Profile/CollectionBadge/CollectionBadge.test.tsx

```diff
@@ -16,7 +16,7 @@ test('renders proper UI when achieved', () => {
 
   expect(screen.getByTestId('CollectionBadge-leverageCrowbar')).not.toBeNull();
   expect(screen.getByText('LEVERAGE CROWBAR')).not.toBeNull();
-  expect(screen.getByText('Achieved: 10/12/22')).not.toBeNull();
+  expect(screen.getByText('Achieved: 02/02/22')).not.toBeNull();
```

Comment:
why is this date hardcoded?

Reply:
the tests are done against a storybook that uses mock data. So we are asserting if the rendering will happen correctly. I am using 02/02 on purpose since I try to format a timestamp into locale of the machine :D (the server where tests are run seems to have different locale than my local one so 02/02, 03/03... etc work)